go/vt/sqlparser/token_test.go

```diff
@@ -189,3 +190,41 @@ func TestSplitStatement(t *testing.T) {
 		}
 	}
 }
+
+func TestSplitStatementToSlice(t *testing.T) {
+	testcases := []struct {
+		input  string
+		output string
```

Comment:
I would define the output here as []string, just to be clear on the actual result. Or use '|' as a separator. So the output of the test is different from the input, otherwise it's hard to tell it does the right thing...